plugins/catalog-import/src/api/CatalogImportClient.ts

```diff
@@ -40,10 +48,12 @@ export class CatalogImportClient implements CatalogImportApi {
   }: {
     repo: string;
   }): Promise<PartialEntity[]> {
+    const idToken = await this.identityApi.getIdToken();
     const response = await fetch(
       `${await this.discoveryApi.getBaseUrl('catalog')}/analyze-location`,
       {
         headers: {
+          authorization: `Bearer ${idToken}`,
```

Comment:
Prolly best to not pass the header if no token is available, just a ``idToken && `Bearer ${idToken}` ``

Otherwise we'll end up throwing around a bunch of `"Bearer "` or `"Bearer undefined"` headers which may cause trouble x)

Reply:
Agreed, this should probably be updated. And not just here, unfortunately, but in the other places as well.